packages/runtime-core/src/renderer.ts

```diff
@@ -882,8 +882,8 @@ function baseCreateRenderer(
             const prev = oldProps[key]
             const next = newProps[key]
             if (
-              next !== prev ||
-              (hostForcePatchProp && hostForcePatchProp(el, key))
+              (hostForcePatchProp && hostForcePatchProp(el, key)) ||
+              next !== prev
```

Comment:
Why do you change this order? Typically a compare operation would be lighter than a method invoke. It's reasonable to have the comparison ahead.

Reply:
yes you are right. I think I should learn more about javascript optimization.
https://jsben.ch/dSVTE